.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 node_modules
+/dist
```

Comment:
To ignore a folder in `.gitignore` the slash goes at the end so in this case it should be:
```
dist/
```

Nonetheless, in this project we want the `dist` folder to be included and public since it's required by npm to publish the plugin. So the actual suggestion is:
```suggestion

```